modules/btcpayserver.nix

```
@@ -0,0 +1,242 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services;
+  inherit (config) nix-bitcoin-services;
+in {
+  options.services = {
+    nbxplorer = {
+      enable = mkEnableOption "nbxplorer";
+      package = mkOption {
+        type = types.package;
+        default = pkgs.nix-bitcoin.nbxplorer;
+        defaultText = "pkgs.nix-bitcoin.nbxplorer";
+        description = "The package providing nbxplorer binaries.";
+      };
+      dataDir = mkOption {
+        type = types.path;
+        default = "/var/lib/nbxplorer";
+        description = "The data directory for nbxplorer.";
+      };
+      user = mkOption {
+        type = types.str;
+        default = "nbxplorer";
+        description = "The user as which to run nbxplorer.";
+      };
+      group = mkOption {
+        type = types.str;
+        default = cfg.nbxplorer.user;
+        description = "The group as which to run nbxplorer.";
+      };
+      btcrpcurl = mkOption {
+        type = types.str;
+        default = "http://127.0.0.1:8332";
+        description = "The RPC server url.";
+      };
+      btcnodeendpoint = mkOption {
+        type = types.str;
+        default = "127.0.0.1:8333";
+        description = ''
+          The p2p connection to a Bitcoin node, make sure you are whitelisted
+        '';
```

```suggestion
        description = "The p2p connection to a Bitcoin node.";
```
The part about whitelisting is a bit irritating, because it doesn't apply in general.
It just refers to a nix-bitcoin implementation detail, namely that our "public" RPC profile isn't sufficient for nbxplorer. But we're already taking care of this with the custom nbxplorer RPC user.

---

I took the description verbatim from `NBXplorer --help`, but I agree with your point. Fixed in 5d150e5